### ISSUE TITLE
Add config option to disable adding metadata on LS -> LS with http plugins

### ISSUE THREAD

**Issue by cstegm** [elastic/logstash#13800]
Hi,

For the Usecase LS -> LS i dont need all the metadata that the http plugin produces. so a config option is needed to switch that off.

**Comment by andsel** [comment]
Hi @cstegm 
the `@metadata` fields are Logstash internal fields so the downstream Logstash instance could leverage those, but doesn't go in your output.
Which is you specific use case to disable the metadata forwarding?

**Comment by cstegm** [comment]
My use case is logstash to logstash communication. I think in this case there ist no reason to generate those fields - so i just want to switch that feature off, if it boots performance a bit.

**Comment by andsel** [comment]
I tried locally with plain HTTP and LS http output plugin doesn't forward upstream's `@metadata`, this is the HTTP payload as seen from Wireshark:
```
POST / HTTP/1.1
Connection: Keep-Alive
Content-Type: application/json
Content-Length: 182
Host: localhost:8090
User-Agent: Manticore 0.8.0
Accept-Encoding: gzip,deflate

{"@version":"1","event":{"original":"{\"name\": \"Andrea\"}","sequence":13},"@timestamp":"2022-04-12T09:42:53.384882Z","host":{"name":"kalispera"},"message":"{\"name\": \"Andrea\"}"}
```

with output config:
```
output {
  http {
    url => "http://localhost:8090"
    http_method => "post"
  }
}
```

Please, could you share the HTTP payload that you see and that shows the `metadata` you refer to?
Because I don't understand which metadata you are referring to.

**Comment by roaksoax** [comment]
I'm going to go ahead and close this issue provided that we have introduced a new way to do Logstash to Logstash communication using `logstash-integration-logstash`. This new plugin provides the functionality for the use of a Logstash Input and Output and will be available by default in 8.11, effectively replacing the use of HTTP Input and HTTP Output.

This new plugin doesn't add metadata to the communication between LS instances. 

More information on https://github.com/elastic/logstash/issues/15163 and https://github.com/elastic/logstash/pull/15346
